helm/defectdojo/templates/configmap.yaml

```diff
@@ -13,7 +13,7 @@ data:
   DD_ADMIN_MAIL: {{ .Values.admin.Mail | default "admin@defectdojo.local" }}
   DD_ADMIN_FIRST_NAME: {{ .Values.admin.FirstName | default "Admin" }}
   DD_ADMIN_LAST_NAME: {{ .Values.admin.LastName | default "User" }}
-  DD_ALLOWED_HOSTS: {{ .Values.host }}
+  DD_ALLOWED_HOSTS: "*"
```

Comment:
what's wrong with secured by default? :)
it worked for me with Values.host, are you sure this change is necessary?

Reply:
I'm using nginx ingress, and when  install  with this parameters, there are errors in log that comes from liveness probles, because it is actually tested by k8s engine.